storybook/index.html

```diff
@@ -9,7 +9,7 @@
             integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs="
             crossorigin="anonymous"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/ramda/0.25.0/ramda.min.js"></script>
-        <script src="/scripts/index.js" type="module" defer></script>
+        <script src="../scripts/index.js" type="module" defer></script>
```

Comment:
the original line should work, as long as you're using a web server (like "go live" in vscode), but let me know if that's not the case